doc/man1/openssl-cmp.pod.in

```diff
@@ -0,0 +1,1142 @@
+
+=item B<-rspin>
+
+Process sequence of CMP responses provided in file(s), skipping server.
+Multiple filenames may be given, separated by commas and/or whitespace.
+As many files are read as needed for the complete transaction.
+
+=item B<-rspout>
+
+Save sequence of CMP responses to file(s).
+Multiple filenames may be given, separated by commas and/or whitespace.
+As many files are written as needed to store the complete transaction.
+
+=item B<-use_mock_srv>
+
+Use the internal mock server for testing the client.
+This works at API level, bypassing HTTP transport.
+
+=back
+
+
+=head2 Certificate verification options, for both CMP and TLS
+
+=over 4
+
+=item B<-policy>, B<-purpose>, B<-verify_name>, B<-verify_depth>,
+B<-attime>,
+B<-ignore_critical>, B<-issuer_checks>,
+B<-policy_check>,
+B<-explicit_policy>, B<-inhibit_any>, B<-inhibit_map>,
+B<-x509_strict>, B<-extended_crl>, B<-use_deltas>,
+B<-policy_print>, B<-check_ss_sig>, B<-crl_check>, B<-crl_check_all>,
+B<-trusted_first>,
+B<-suiteB_128_only>, B<-suiteB_128>, B<-suiteB_192>,
+B<-partial_chain>, B<-no_alt_chains>, B<-no_check_time>,
+B<-auth_level>,
+B<-allow_proxy_certs>
+
+Set various options of certificate chain verification.
+See L<openssl(1)/Verification Options> for details.
+
+=back
+
+
+=head2 Mock server options, for testing purposes only
+
+=over 4
+
+=item B<-port>
+
+Act as CMP HTTP server mock-up listening on the given port.
+
+=item B<-max_msgs>
+
+Maximum number of CMP (request) messages the CMP HTTP server mock-up
+should handle, which must be non-negative.
+The default value is 0, which means that no limit is imposed.
+In any case the mock server terminates on internal errors, but not when it
+detects a CMP-level error that it can successfully answer with an error message.
+
+=item B<-srv_ref>
+
+Reference value to use as senderKID of server in case no B<-srv_cert> is given.
+
+=item B<-srv_secret>
+
+Password source for server authentication with a pre-shared key (secret).
+
+=item B<-srv_cert>
+
+Certificate used by the server.
+
+=item B<-srv_key>
+
+Private key used by the server for signing messages.
+
+=item B<-srv_keypass>
+
+Server private key (and cert file) pass phrase source.
+
+=item B<-srv_trusted>
+
+Trusted certificates for client authentication.
+
+=item B<-srv_untrusted>
+
+Intermediate certs for constructing chains for CMP protection by client.
+
+=item B<-rsp_cert>
+
+Certificate to be returned as mock enrollment result.
+
+=item B<-rsp_extracerts>
+
+Extra certificates to be included in mock certification responses.
+
+=item B<-rsp_capubs>
+
+CA certificates to be included in mock ip response.
+
+=item B<-poll_count>
+
+Number oftimes the client must poll before receiving a certificate.
+
+=item B<-check_after>
+
+The checkAfter value (time to wait) to include in poll response.
+
+
+=item B<-grant_implicitconf>
+
+Grant implicit confirmation of newly enrolled certificate.
+
+=item B<-pkistatus>
+
+PKIStatus to be included in server response.
```

Comment:
What are the allowed values?

Reply:
I've added this info, also for other cases below.